### DISCUSSION TITLE
It's possible to accidentally merge a draft pull request

### DISCUSSION THREAD

**Discussion by Alexandria-Davis-Ametek** [community/community#56271] [Pull Requests]
### Select Topic Area

Bug

### Expected behavior:
Once a pull request is marked as draft, it should not be possible to merge it.

### Actual behavior:
If you have a pull request open and the pull request is changed to a draft, you can still merge it.

### Steps to reproduce:

* Create a pull request on computer A using one account
* Open the pull request on computer B, and approve the changes with another account
* Return to computer A and change the pull request to a draft
* Merge the pull request on computer B

**Comment by maitrid** [comment]
Just observed a different sequence that also caused an accidental merge, still needing to be reproduced:

1. Create a draft pull request for branch A
2. The owner of branch B rebased from master (or whatever common target branch name)
3. Branch A got included in the commit of branch B as a contribution (author of branch A becomes a co-author of branch B)
4. Pull request for branch B is approved
5. The pull request for branch B is merged along with changes in branch A
